docs/en/observability/whats-new.asciidoc

```diff
@@ -14,4 +12,165 @@ coming[8.0.0]
+
+For more information, see our
+https://www.elastic.co/blog/elastic-and-aws-serverless-application-repository-speed-time-to-actionable-insights-with-frictionless-log-ingestion-from-amazon-s3[blog]
+and https://github.com/elastic/elastic-serverless-forwarder/blob/main/docs/README-AWS.md[documentation].
+
+[discrete]
+== Custom AWS logs integration for {agent}
+
+We have added a new Custom AWS logs integration that enables you to ingest any
+custom logs from the AWS environment for which we don’t have an existing
+out-of-the-box {agent} integration. This new integration supports both
+Amazon S3 and CloudWatch logs as an input source for these logs. You can choose
+to set up your own index templates and ingest pipelines, and configure the
+integration to write data to a specific data stream. You can then use the power
+of {kib} to analyze the collected data.
+
+[role="screenshot"]
+image::images/integrations-custom-aws-logs.png[Add Customer AWS Logs integration]
+
+[discrete]
+== {agent} Azure virtual machine extension
+
+The new Microsoft Azure VM extension for {agent} provides Azure users a
+very simple way to install and register {agent} on all their Azure VM’s.
+It automatically installs {agent}, registers it with {fleet}, creates a
+separate policy that collects system logs and metrics, and assigns this policy
+— all with a single checkbox or command line option.
+
+You can use the Elastic deployment pane on the Azure console to configure
+Elastic Azure VM extension with a single click. If you prefer to manage with
+scripts, you can use Azure CLI or Powershell to interact with the extension and
+get {agent} installed and configured on all your Azure VM’s.
+
+[role="screenshot"]
+image::images/azure-elastic-vm.png[{es} VM extension installed on Azure]
+
+Once configured you can manage the lifecycle of your {agents} from within
+{fleet}. For more information, see the https://www.elastic.co/guide/en/cloud/current/ec-azure-marketplace-native.html[Native Azure integration documentation]
+and https://www.elastic.co/guide/en/observability/current/monitor-azure.html[Monitor Microsoft Azure tutorial].
+
+[role="screenshot"]
+image::images/fleet-azure-vm-policy.png[{agent} policy for Azure VM]
+
+[discrete]
+== {agent} support for {ls} output (public beta)
+
+You can now add a {ls} instance as an output to an {agent} policy.
+This feature is available as a public beta, with the goal of making it generally
+available (GA) in 8.4. Inline instructions are provided to Configure {ls}
+for {agent} (as shown below) that step through the requirements to
+establish a secure connection between agents and the {ls} instance. Please
+note that mutual TLS is enforced when agents are set to write to a {ls} output.
+
+[role="screenshot"]
+image::images/fleet-add-logstash-output.gif[Add new {ls} output via {fleet}]
+
+[discrete]
+== APM support for {kib} spaces
+
+You can now configure APM UI indices in the context of {kib} spaces. This
```

Comment:
I think "APM UI indices" is too condensed. Makes it sound like the indices are only valid when used with the UI. If @bmorelli25 thinks it's Ok as-is, I will back off.

Reply:
I think this is OK as-is. This feature allows users to customize the indices that the APM UI queries—so it is only valid when used with the UI.